srcpkgs/libircclient/template

```diff
@@ -0,0 +1,54 @@
+# Template file for 'libircclient'
+pkgname=libircclient
+version=1.10
+revision=1
+build_style=gnu-configure
+configure_args="--enable-openssl --enable-ipv6 --enable-shared"
+makedepends="libressl-devel"
+short_desc="IRC client library"
+maintainer="maciozo@maciozo.com <maciozo@maciozo.com>"
+license="LGPL-3.0-or-later"
+homepage="https://www.ulduzsoft.com/linux/libircclient/"
+distfiles="https://netix.dl.sourceforge.net/project/libircclient/libircclient/${version}/libircclient-${version}.tar.gz"
+checksum=bbb26f3af348b252c5204917a7f91cfdf172f1b6afbf4df1e561b03e20503c2d
+
+case "$XBPS_TARGET_MACHINE" in
+	*-musl)	depends+=" musl-devel";;
+	*)		depends+=" glibc-devel";;
+esac
+
+do_install() {
+	mkdir -p ${PKGDESTDIR}/usr/lib
+	mkdir -p ${PKGDESTDIR}/usr/include
+	mkdir -p ${PKGDESTDIR}/doc/sources
+	cp ${wrksrc}/src/*.so ${PKGDESTDIR}/usr/lib/
+	cp ${wrksrc}/src/*.a ${PKGDESTDIR}/usr/lib/
+
```

Comment:
Read the Manual.md for how to use vcopy, vbin, vmove etc.
This saves you from manually creating directories in the `$DESTDIR` (main install) or `$PKGDESTDIR` (subpackages).